### ISSUE TITLE
Extended Cyrillic?

### ISSUE THREAD

**Issue by Uranbold** [floriankarsten/space-grotesk#4]
Awesome. Do you consider about the adding the Cyrillic Extended?

Thanks.

**Comment by mightybart** [comment]
Well, in very very distant future, maybe 🤔 there were some requests and possible commission, but can't promise anything right now.

**Comment by sergeevabc** [comment]
@mightybart, why have you closed the issue? Cyrillic is not added yet.

**Comment by mightybart** [comment]
True, maybe I should've left it open. But my answer to the original question is "No, I do not consider adding Cyrillic". I leave this open, to anyone who would like to contribute.

**Comment by davelab6** [comment]
V3 issue :)

**Comment by mightybart** [comment]
V3 is going to be huge! 😄

**Comment by Uranbold** [comment]
How can i help on this project? I can help with anything...

**Comment by ravid-dubsteper** [comment]
![image](https://user-images.githubusercontent.com/78262241/174627591-a6a01237-0bc6-45fc-8017-f0f52c7a82c9.png)
👀

**Comment by n1ghtmare** [comment]
@ravid-dubsteper Any progress on this? Looks awesome!

**Comment by ravid-dubsteper** [comment]
@n1ghtmare hello. thank u :)
basic cyrillic is complete, but it does not have kerning. also it does not have an extended character set for cyrillic, alts for cyrillic and bulgarian and serbian localizations.

**Comment by n1ghtmare** [comment]
@ravid-dubsteper I don't know anything about font design, I wish I could be of any help. Let me know if there is anything I can do!

**Comment by ravid-dubsteper** [comment]
@n1ghtmare idk
everything I said above has to do with type design :))

**Comment by ravid-dubsteper** [comment]
so here is my progress for now.
i designed the basic cyrillic set, made ss01 (alt a), ss03 (alt y) and ss04 (alt Б and В ) for basic cyrillic. also i changed the form of the б a bit
![image](https://user-images.githubusercontent.com/78262241/211107831-adb2fe8a-5051-4585-a442-2061f9207e0a.png)
![image](https://user-images.githubusercontent.com/78262241/211107867-d912b3d0-ed43-4bf9-a487-d148475c2d5c.png)

**Comment by victormihalache** [comment]
@ravid-dubsteper looks amazing!

**Comment by victormihalache** [comment]
@ravid-dubsteper out of curiosity, what still remains to be done before this can be potentially merged?

**Comment by ravid-dubsteper** [comment]
@victormihalache i created the request with basic cyrillic set here: #50
